landing-pages/site/layouts/partials/footer.html

```diff
@@ -63,3 +63,8 @@
         </span>
     </div>
 </footer>
+
+<!-- Scroll to Top Button -->
+<button id="scroll-to-top" class="scroll-to-top" aria-label="Scroll to top">
+    <span class="scroll-to-top__arrow"></span>
+</button>
```

Comment:
I think it’s strange for this code to be placed below the footer. Couldn’t we implement it in the same way as the ‘Suggest a change on this page’ button?

Reply:
Nice. Looking forward